service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/AbstractDocmosisOrder.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.groupingBy;
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+
+@Getter
+@AllArgsConstructor
+public class AbstractDocmosisOrder extends AbstractDocmosisData {
```

Comment:
Are we sure all of those are true for any order? I.e. we don't have complianceDeadline for most of the order do we? It seems only for SDO / CMO, might be confusing if we wanted to reuse this class for other orders. Would you consider renaming?

Reply:
No these are not all common for orders, I've already had this chat with Lewis and I going to keep working on as I work my way through other orders. This Order object will probably turn into something a little more specific to group is as CaseManagementOrders maybe? And AbstractDocmosisOrder will keep whatever is common.